test/parallel/test-tls-async-cb-after-socket-end.js

```diff
@@ -1,66 +1,67 @@
 'use strict';
-
 const common = require('../common');
 if (!common.hasCrypto)
   common.skip('missing crypto');
 const fixtures = require('../common/fixtures');
 const SSL_OP_NO_TICKET = require('crypto').constants.SSL_OP_NO_TICKET;
 const tls = require('tls');
 
+// Check tls async callback after socket ends
+
 const options = {
   secureOptions: SSL_OP_NO_TICKET,
   key: fixtures.readSync('test_key.pem'),
   cert: fixtures.readSync('test_cert.pem')
 };
 
-const server = tls.createServer(options, function(c) {
-});
+const server = tls.createServer(options, common.mustCall((c) => {
```

Comment:
Since the callback is empty it can be omitted from `common.mustCall()`.

Reply:
fixed

Reply:
I meant that this could be just `common.mustCall()`.

Reply:
sorry wrong interpreted

Reply:
change made :)